extension/package.json

```diff
@@ -82,14 +77,16 @@
     "postcss-scss": "^3.0.2",
     "prettier": "^1.11.0",
     "remotedev-server": "^0.3.1",
-    "selenium-webdriver": "^3.6.0",
+    "sanitize-filename": "^1.6.3",
+    "selenium-webdriver": "^4.0.0-beta.1",
     "source-map-loader": "^0.2.4",
     "source-map-support": "^0.5.19",
     "style-loader": "^1.1.3",
     "svg-url-loader": "^5.0.0",
     "tailwindcss": "^1.1.3",
     "trash-cli": "^1.4.0",
     "ts-loader": "^6.2.2",
+    "ts-mocha": "^8.0.0",
     "tslint": "^5.11.0",
```

Comment:
Per [tslint homepage](https://www.npmjs.com/package/tslint):

> TSLint has been deprecated in favor of ESLint. Please see `https://github.com/palantir/tslint/issues/4534` for more information.

I haven't tried it yet, but I did see [tslint-to-eslint-config](https://www.npmjs.com/package/tslint-to-eslint-config) in my feed this morning and it might be useful.

Reply:
Thanks @pdehaan, this is a useful push for me to remove tslint as well as part of this PR :)

Reply:
This is now done